### ISSUE TITLE
JQuery update planned?

### ISSUE THREAD

**Issue by jeff89179** [ytti/oxidized#3295]
Hi,
We recently ran a Nessus scan in our environment and the following vulnerability was tagged for the Oxidized container we're running.

JQuery 1.2 < 3.5.0 Multiple XSS (Medium Severity)
https://nvd.nist.gov/vuln/detail/CVE-2020-11022
https://nvd.nist.gov/vuln/detail/CVE-2020-11023

Nessus' solution is to update to at least 3.5.0. 

Is there any plan to update the version of JQuery used?

**Comment by robertcheramy** [comment]
1. This is an oxidized-web issue. Open it there.
2. You are not using the latest oxidized-web. It ships with [jQuery 3.7.1](https://github.com/ytti/oxidized-web/blob/147fbee7ec694adac6bf56f92d52ad531a10dc54/package.json#L19)

**Comment by jeff89179** [comment]
Would that be in the latest docker image from oxidized/oxidized:latest?

**Comment by robertcheramy** [comment]
Yes, you are right, the latest stable oxidized container has still an old oxidized-web. This will be solved with next release.
